appserver/tests/quicklook/web/helloworld/src/test/HelloJSPTestNG.java

```diff
@@ -115,34 +116,23 @@ public void simpleJSPTestPage() throws Exception{
 
     }
 
-    @Test(groups={"pulse"}) //test method for server
-    public void testServerRunning() throws Exception{
-	    //Your server is up and running!
-	    //
-	String testurl = "http://" + host  + ":" + port;
+    @Test(groups = {"pulse"}) //test method for server
+    public void testServerRunning() throws Exception {
+        //Your server is up and running!
+        //
+        String testurl = "http://" + host + ":" + port;
         //System.out.println("URL is: "+testurl);
         URL url = new URL(testurl);
         //echo("Connecting to: " + url.toString());
         HttpURLConnection conn = (HttpURLConnection) url.openConnection();
         conn.connect();
         int responseCode = conn.getResponseCode();
+        boolean result = false;
```

Comment:
These changes are not required

Reply:
yes those are required to pass the tests because when reviewing I saw that the content that is added to the inputstream now is empty. That is why I{m just comparing the code of the response that must be 200